frontend/src/Components/SwapCoin/Coins.js

```diff
@@ -0,0 +1,183 @@
+const coins = [
+    {"id": "btc-bitcoin",
+    "symbol": "BTC",
+    "name": "Bitcoin", 
+    "image": "https://cdn.pixabay.com/photo/2015/08/27/11/20/bitcoin-910307_960_720.png", 
+    "price": 57727.46216266, 
+    "rank": 1, "marketcap": 1077256500977, 
+    "price_24h_percentage_change": 2.47, 
+    "price_7d_percentage_change": 2.62, 
+    "ticker_price_24h": [
+        56191.244999999995, 
+        56514.09999999999, 
+        57125.40375, 
+        57282.07, 
+        57344.557499999995, 
+        57393.579999999994, 
+        57309.89625, 
+        57291.835, 
+        57303.666249999995, 
+        57379.54125000001, 
+        57572.07375, 
+        57605.1, 
+        57868.4025, 
+        57850.8725, 
```

Comment:
We're supposed to fetch these tho right?

Reply:
Yeah. This is dummy data while waiting for endpoints from the backend